.vsts-ci.yml

```diff
@@ -109,11 +109,17 @@ jobs:
   pool:
     vmImage: 'vs2017-win2016'
   strategy:
-    maxParallel: 4
+    maxParallel: 5
     matrix:
-      r_package:
+      r_package_msvc:
         TASK: r-package
         COMPILER: MSVC
+        R_VERSION: 3.6
```

Comment:
I think it makes sense to mix R_VERSION: 3.6 for one job and 4.0 for another one. And the same thing for Appveyor.

Reply:
ok

Reply:
changed in https://github.com/microsoft/LightGBM/pull/3065/commits/c65a91078d23001962a956abfc915ad7ebfba155